.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,50 +1,36 @@
-<!--- Before submitting your pull request, --->
-<!--- please complete as much as practicable of the following checklist: --->
+<!--- Type an x in the box [x] before submitting --->
```

Comment:
I think this is not needed here anymore.

Reply:
:+1: